includes/Abstracts/AbstractWalletBlock.php

```diff
@@ -34,6 +33,7 @@ public function get_payment_method_data(): array {
 			'total_limitation' => $settings->getWidgetPaymentWalletMinMax( $payment ),
```

Comment:
@andryes  @FabianaRBarros Just FYI I don't know if it's correct but in the `power_board` branch this method is not present: `getWidgetPaymentWalletMinMax`

Reply:
This is something that was present on `main` branch and not on `powerboard`. I check and it was added by jsp [here](https://github.com/PayDock/e-commerce-woo/commit/37d99e953b4646579f5fdf793a36931049cc62c1), so I assumed it should not be removed as it was a recent implementation.

Reply:
Maybe @andryes and @jpcostello have more information about this. Should we keep or remove?

Reply:
Yes, or maybe we should add this to the `power_board` :)

Reply:
@FabianaRBarros @Danyl-Kushnaryov This line can be removed - min/max checks should only take place for APMs (AbstractAPMsBlock)